src/main/java/de/dhbw/karlsruhe/domain/services/dialogs/MenuDialogService.java

```diff
@@ -64,6 +65,8 @@ private int awaitUserInput() {
             } catch (InputMismatchException ie) {
                 outputPort.optionError();
                 inputPort.cleanInput();
+            } catch (InvalidOptionException ioe) {
+                outputPort.optionError();
```

Comment:
Warum bei exception davor noch cleanInput() und hier nicht?

Reply:
Könnte mir vorstellen zur besseren Nachvollziehbarkeit für den Anwender...
Aber ich denke Marco kann uns aufklären :)

Reply:
Kann es nicht mehr genau sagen..aber wenn man in den InputMismatch Fehler reinläuft, gibt es mit dem Scanner Probleme und dadurch auch mit der folgenden Eingabe. Also Fix kann man eine Zeile einlesen. Somit kommt es nicht zu Probleme. Das hatte ich mal eingebaut, weil wir da dann auch einen Bug hatten. Im Fall der InvalidOptionException ist dies aber nicht nötig.